packages/api/src/routers/api/alerts.ts

```diff
@@ -71,6 +99,45 @@ const validateGroupBy = async (req, res, next) => {
 };
 
 // Routes
+router.get('/', async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    if (teamId == null) {
+      return res.sendStatus(403);
+    }
+    const alerts = await Alert.find({ team: teamId });
+    const alertsWithHistory = await Promise.all(
+      alerts.map(async alert => {
+        const history = (await getHistory(alert, teamId.toString())) ?? [];
+        if (!alert.source) throw new Error('Alert source is undefined');
+        if (alert.source === 'LOG') {
+          const logView = await getLogView(alert, teamId.toString());
+          // had to rename because logView is an ObjectID
+          return {
+            logViewObj: logView,
```

Comment:
I assume `logView` is the redundant info. why don't we overwrite `logView` with object here ?